h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -22,6 +19,11 @@
 import java.io.IOException;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Consumer;
```

Comment:
Java8 specific - failing build directly - try it locally with Java7 and `./gradlew build`